rollup.config.js

```diff
@@ -34,6 +34,10 @@ function serve() {
 }
 
 export default {
+  onwarn: (warning, handler) => {
+    if (warning.message.indexOf("Unused CSS selector") !== -1) return;
+    handler(warning);
+  },
```

Comment:
This is strictly unrelated to this PR, but prevents the console from filling up with warnings while we buid the project.

Reply:
This is great! We also do something similar in storybook https://github.com/mozilla/glean-dictionary/blob/8d9a6a2b6aed9648c025210ee2ca30066b54165f/.storybook/main.js#L32-L36